frontend-ssr/components/cli_git/match_commands.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+
+// class MatchWords extends React.Component {
+//     constructor(props) {
+//         super(props);
+
+//         this.state = {
+//             message: '',
+//         }
+//     }
+
+//     handleSubmit = (event) => {
+//         event.preventDefault();
+
+//         const inputValue = document.querySelector('[name="input_text"]').value;
```

Comment:
Let's not use `querySelector` on the `document`. It's highly ineffective searching the entire DOM. You can get directly to the input from the `event`, no querySelectors.

`event.target['input_text'].value` should work